packages/astro/components/ViewTransitions.astro

```diff
@@ -30,6 +30,21 @@ const { fallback = 'animate' } = Astro.props as Props;
 		!!document.querySelector('[name="astro-view-transitions-enabled"]');
 	const triggerEvent = (name: Events) => document.dispatchEvent(new Event(name));
 	const onPageLoad = () => triggerEvent('astro:page-load');
+	const announce = () => {
+		let div = document.createElement('div');
+		div.setAttribute('aria-live', 'assertive');
+		div.setAttribute('aria-atomic', 'true');
+		div.setAttribute('style', 'position:absolute;left:0;top:0;clip:rect(0 0 0 0);clip-path:inset(50%);overflow:hidden;white-space:nowrap;width:1px;height:1px');
```

Comment:
Maybe let's leave a comment that tells us what this style does and why. +1 for a link that explains it.